assets/css/style.css

```diff
@@ -1334,3 +1334,9 @@ footer a:hover{
         display: none !important;
     }
 }
+
+@media (max-width: 350px) {
+    .globalLogoImg{
+        width:170px !important;
```

Comment:
Before, I know I said we could use a media query… but in reflection, it is not a best practice to create a single CSS rule that is only used in one, specific place. One thing that is nice about Bootstrap is that it provides [containers](https://getbootstrap.com/docs/4.6/layout/overview/) with breakpoints pre-defined and handled in Bootstrap itself…

@Neha9849 Could you dig into this one and see if we can eliminate the custom CSS by instead using a container, perhaps a standard/small one? I'd love to see this change implemented without creating new CSS.

Later down the road, this will make it easier for us to upgrade to Bootstrap v5. This would bring some added benefits and allow us to refactor existing code based on new changes introduced in Bootstrap v5. Currently we are using v4.6 if I recall correctly. I hope this feedback makes sense! I know it is trickier and not what we discussed, but I was looking at our code and I hoped to make the future upgrade work less cumbersome.

Reply:
I will remove the PR in this and will make the CSS fixes in other as this PR is already very prolonged and it would be nice to have this fix before user interviews.